process-engine/process_engine_runtime
# ✨ Trigger TimerEvents with past date values immediately

## Changes

1. Immediately start TimerEvents that use a date-type timer with a past date as value.
2. Robustify timer disposal for each handler that uses them. 

## Issues

Closes #337

PR: #339

## How to test the changes

Best use this Runtime version with the BPMN studio to test this.

- Create a ProcessModel that uses a date-type TimerEvent with a past date as value
- Execute that ProcessModel
- See that the timerEvent gets triggered immediately
    - Note that a warning will be displayed in the console, notifying the user that the date he used is not optimal
